chpl/chpl-service/src/main/java/gov/healthit/chpl/user/cognito/CognitoApiWrapper.java

```diff
@@ -371,6 +367,31 @@ public void updateUser(User user) throws UserRetrievalException {
         cognitoClient.adminUpdateUserAttributes(request);
     }
 
+    @CacheEvict(value = CacheNames.COGNITO_USERS, key = "#user.cognitoId")
```

Comment:
It looks like we cache users both with UUID as a key and email as a key (see above method `getUserInfo(String email)`). But here we are evicting only the UUID key. I'm not 100% sure I'm correct but I think so... can you check on this?